src/utils/parseQueryParams.js

```diff
@@ -0,0 +1,46 @@
+export function parseQueryParams() {
+  var queryString = window.location.search;
+  if (queryString === '') return {};
+  var keyValPairs = [];
+  var params = {};
+  queryString = queryString.replace("?", "");
+  queryString = queryString.replace(/%20/g, " ");
+  queryString = queryString.replace(/%2C/g, ",")
+
+  if (queryString.length) {
```

Comment:
Is this necessary? You've already checked this with `if (queryString === '') return {};` in line 3